src/components/backend-ai-session-list.ts

```diff
@@ -326,11 +326,11 @@ export default class BackendAiSessionList extends BackendAIPage {
   }
 
   get _isRunning() {
-    return this.condition === 'running';
+    return ['batch', 'interactive', 'running'].includes(this.condition);
   }
 
   _isPreparing(status) {
-    const preparingStatuses = ['RESTARTING', 'PREPARING', 'PULLING'];
+    const preparingStatuses = ['RESTARTING', 'PREPARING', 'PULLING', 'PENDING'];
```

Comment:
Is there a reason to add the `PENDING` status to the `preparingStatuses`?

Reply:
Nope, I just added it to make sure the log icon to be disabled in `PENDING` status, but I found that it doesn't need to be added.